09-secrets.md

```diff
@@ -1,28 +1,33 @@
 ## Secrets top-level element
 
-Secrets are a flavour of Configs focusing on sensitive data, with specific constraint for this usage. As the platform implementation may significantly differ from Configs, dedicated Secrets section allows to configure the related resources.
+Compose provides a way for you to use secrets without having to use environment variables to store information. If you’re injecting passwords and API keys as environment variables, you risk unintentional information exposure. Environment variables are often available to all processes, and it can be difficult to track access. They can also be printed in logs when debugging errors without your knowledge. Using secrets mitigates these risks.
 
-The top-level `secrets` declaration defines or references sensitive data that can be granted to the services in this
-application. The source of the secret is either `file` or `external`.
+Secrets are also useful for locally replicating a remote development setup with Kubernetes. Using secrets makes your application scalable.
```

Comment:
how do compose secrets help "replicating  a remote development setup with Kubernetes" ?
AFAICT kube secrets are not exact equivalent for docker swarm nor compose secrets

Also, "Using secrets makes your application scalable" doesn't make sense to me. Scalability isn't involved here

Reply:
From my conversation with Milas, my understanding was that these were two additional benefits of using Secrets. But perhaps I misunderstood. Is there something else I can put here? What would you recommend? Or is it as simple as Secrets are superior to environment variables when storing/using sensitive information?

Reply:
There's no other benefits that comes to my mind